### PR TITLE
Bugfix/close modal when update

### PR DESCRIPTION
**Pull request by weslybrandao**
#### What problem is this solving?

Before this adjustment, the modal did not close when the url changed

#### How to test it?

You can see in my example that when you click on a link the modal closes

[Workspace](https://mcdon28--devmcdbrdeliveryio.myvtex.com/catalog)

#### Screenshots or example usage:

Use the CEP: 01020-000

First step open modal 

image
 
and finnaly click to your option

image

#### How does this PR make you feel? [:link:](http://giphy.com/)

Happy to be able to create new things

happy

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ Now, you are able to use all blocks exported by the `modal-layout` app. Check ou
 | `fullScreen` | `boolean` | Whether the modal should fill the whole screen (`true`) or not (`false`). This prop is responsive i.e. it adapts itself to the device's breakpoints.  | `false` | 
 | `backdrop` | `enum` | Whether the modal will have a clickable backdrop (`clickable`) or no backdrop at all (`none`). This prop is responsive i.e. it adapts itself to the device's breakpoints. | `clickable` | 
 
+
```
Comment (guerreirobeatriz) [inline docs/README.md:98]:
```suggestion
```

**Review by guerreirobeatriz — APPROVED** [review]

**Comment by klzns** [comment]
Tested the new code at https://brenovtex--devmcdbrdeliveryio.myvtex.com/ using CEP 01020-000

**Review by klzns — APPROVED** [review]

**Review by kaisermann — APPROVED** [review]

**Review by Klynger — APPROVED** [review]
Hey @weslybrandao, thank you for this PR! This idea is really good and is going to help us remove some dependencies that we made to close modals when something happens (that were usually URL changes)

**Comment by klzns** [comment]
@allcontributors add @weslybrandao for code

**Pull request merged**